### PR TITLE
Update benchmarks

### PR DESCRIPTION
**Pull request by IronGauntlets**
Please view each commit separately.

### CODE REVIEW

File: core/crypto/ecdsa_test.go [hunk 1]
```diff
@@ -57,6 +57,8 @@ func TestVerify(t *testing.T) {
 	}
 }
 
+var benchVerifyR bool
```
Comment (pnowosie) [inline core/crypto/ecdsa_test.go:60]: Why R? I'm curious about its origin - perhaps it's time for a creative renaming! Any suggestions for what R could stand for? (Wrong answers only, of course!)

Comment (IronGauntlets) [reply core/crypto/ecdsa_test.go:60]: R is for Result.

Comment (pnowosie) [inline core/crypto/ecdsa_test.go:60]:
```suggestion
var benchVerifyResult bool
```

**Review by pnowosie — APPROVED** [review]
My only consern is rename `*R -> *Result` if possible.
Thanks

File: core/crypto/ecdsa_test.go (hunk 1 above)
Comment (IronGauntlets) [reply core/crypto/ecdsa_test.go:60]: I think it is fine as is.

**Review by AnkushinDaniil — APPROVED** [review]

**Pull request merged**